src/server/modules/user/UserController.ts

```diff
@@ -66,6 +67,7 @@ export class UserController {
         longUrl,
         file,
       )
+      dogstatsd.increment('shortlink.create', 1, 1, [`isfile:${!!file}`])
```

Comment:
Should we move this to the service layer? If this line is called somewhere else, caller might have to duplicate the datadog stat line as well

Reply:
makes sense, i'll move it down to url management service!